scripts/deploy.backend.sh

```diff
@@ -14,18 +14,67 @@ case $ENV in
     ;;
 esac
 
+II_CANISTER_ID="$(dfx canister id internet_identity --network "$ENV")"
+POUH_ISSUER_CANISTER_ID="$(dfx canister id pouh_issuer --network "$ENV")"
+# URL used by II-issuer in the id_alias-verifiable credentials (hard-coded in II)
+# Represents more an ID than a URL
+II_VC_URL="https://identity.ic0.app"
+# URL used by issuer in the issued verifiable credentials (tipically hard-coded)
+# Represents more an ID than a URL
+# TODO: To be confirmed by DECIDE AI team
+POUH_ISSUER_VC_URL="https://www.elna.ai"
+
+
+if [[ "$ENV" = "local" || -z "${ENV}" ]]; then
```

Comment:
For readability reason, I would move the initialization of any variable depending of the environment in the existing `case` block?

Reply:
Yes, good point.